frame/nomination-pools/rpc/runtime-api/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
```

Comment:
This folder should be moved to `frame/nomination-pools/runtime-api/Cargo.toml`, no longer the `rpc` folder is needed.

Sorry, last comment 😅

Reply:
Ok, I thought that we might create RPC for this in the future. I am going to delete it now :)